### ISSUE TITLE
[Feature proposal] Gradient hue on pairplot

### ISSUE THREAD

**Issue by GuillaumeDesforges** [mwaskom/seaborn#1591]
Hello,

In most data exploration, the user wants to see how two dimensions are related to one each other, plus how this relationship can correlate with another parameter. This is possible in seaborn thanks to `sns.pairplot`, and using the `hue` parameter when we want to spot whenever two dimensions can help in a clustering task.

In the case where we want to do a continous prediction, the current workaround is to split the target dimension of the prediction into bins and use `hue`, but it would be way nicer and visual if one could have each plot color correspond to the color of the target value of said point in a gradient.

In short: would it be possible to have the color of the plots in relative/pair plotting in a gradient way based on a continuous dimension?

Thanks,
Guillaume

**Comment by mwaskom** [comment]
Not easily, sorry.

**Comment by mwaskom** [comment]
It was indeed not easy, but https://github.com/mwaskom/seaborn/pull/2234

**Comment by GuillaumeDesforges** [comment]
Awesome! Good work
